src/Sylius/Behat/Client/Request.php

```diff
@@ -211,4 +211,13 @@ private function mergeArraysUniquely(array $firstArray, array $secondArray): arr
 
         return $firstArray;
     }
+
+    private static function prepareSection(?string $section): string
```

Comment:
this function should be deleted after refactor all new API endpoints, for now only refactored endpoint have `section` prefix, without this logic a have problems with pathnames while testing.

Reply:
IMHO, we can leave it for the future, to give possibility of using this class without any section

Reply:
I will fix it in next PR